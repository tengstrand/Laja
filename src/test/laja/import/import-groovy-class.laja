
#import "src/test/laja/import/TestClass.groovy"

#set (testClass = new TestClass())

#write "target/test-output/import-groovy-class.out"
increment(1) = {testClass.increment(1)}
#end
