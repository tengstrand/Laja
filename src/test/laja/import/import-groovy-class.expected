increment(1) = 2
